addons/actions/src/components/ActionLogger/index.tsx

```diff
@@ -6,11 +6,12 @@ import { withTheme } from '@storybook/theming';
 import { ActionBar, ActionButton } from '@storybook/components';
 
 import { Actions, Action, Wrapper, InspectorContainer, Countwrap, Counter } from './style';
+import { ActionDisplay } from '../../models';
 
-const ActionLogger = withTheme(({ actions, onClear, theme }) => (
+export const ActionLogger = withTheme(({ actions, onClear, theme }: any) => (
```

Comment:
Why are we moving from default to named exports?

Reply:
Mainly because named exports are easier to refactor and maintain than default ones. There are few others reasons explained in TypeScript Deep Dive: https://basarat.gitbooks.io/typescript/content/docs/tips/defaultIsBad.html

Reply:
I agree